This patch is to integrate with the package namespace for Debian and derived
distributions.  It has not gone, and does not need to go, upstream.
Index: pypolicyd-spf-2.0.0/policyd-spf
===================================================================
--- pypolicyd-spf-2.0.0.orig/policyd-spf	2016-12-02 01:23:29.363567669 -0500
+++ pypolicyd-spf-2.0.0/policyd-spf	2016-12-02 01:23:29.355567669 -0500
@@ -738,7 +738,7 @@
 ###################################################
 #  load config file  {{{1
 #  Default location:
-configFile = '/etc/python-policyd-spf/policyd-spf.conf'
+configFile = '/etc/postfix-policyd-spf-python/policyd-spf.conf'
 if len(sys.argv) > 1:
     if sys.argv[1] in ( '-?', '--help', '-h' ):
         print('usage: policyd-spf [<configfilename>]')
Index: pypolicyd-spf-2.0.0/policyd-spf.1
===================================================================
--- pypolicyd-spf-2.0.0.orig/policyd-spf.1	2016-12-02 01:23:29.363567669 -0500
+++ pypolicyd-spf-2.0.0/policyd-spf.1	2016-12-02 01:26:49.483572313 -0500
@@ -145,12 +145,13 @@
 
 $ policyd-spf \-h (Display usage message)
 
-$ policyd-spf /etc/policyd-spf/policyd-spf.conf (Config file name to use)
+$ policyd-spf /etc/postfix-policyd-spf-python/policyd-spf.conf
+  (Config file name to use)
 
-Configuration options are described in the sample configuration file provided
-with the package (policyd-spf.conf.commented) and in policyd-spf.conf(5).  The
-provided setup.py installs an uncommented configuration file in 
-/etc/policyd-spf/.
+Configuration options are described in the sample configuration file installed
+in /usr/share/doc/postfix-policyd-spf-python (policyd-spf.conf.commented) and
+in policyd-spf.conf(5).  The provided setup.py installs an uncommented
+configuration file in /etc/postfix-policyd-spf-python/.
 
 Additionally, whitelisting certain IP addresses or IP addresses used by listed
 domains from SPF checks is supported.  Skipping SPF checks for local submission
@@ -254,15 +255,13 @@
  1. Add the following to /etc/postfix/master.cf:
 
         policyd-spf  unix  -       n       n       -       0       spawn
-            user=nobody argv=/usr/bin/policyd-spf
+            user=policyd-spf argv=/usr/bin/policyd-spf
 
-    NOTE: Check the path to both the installed Python 3 interpreter and
+    NOTE: Check the path to both the installed Python interpreter and
           policyd-spf.  These vary from system to system.  To use non-default
-          settings, you must also add the config file (see above and 
-          policyd-spf.conf(5) for details).  If you run other services with
-          user nobody, create a dedicated user for this policy server and use
-          that instead.  Python and Python 3 versions prior to 3.3 are not
-          supported.
+          settings, you must also add the config file (see above and
+          policyd-spf.conf(5) for details).  Python and Python 3 versions
+          prior to 3.3 are not supported.
 
  2. Configure the Postfix policy service in /etc/postfix/main.cf:
 
Index: pypolicyd-spf-2.0.0/policyd-spf.conf.5
===================================================================
--- pypolicyd-spf-2.0.0.orig/policyd-spf.conf.5	2016-12-02 01:23:29.363567669 -0500
+++ pypolicyd-spf-2.0.0/policyd-spf.conf.5	2016-12-02 01:23:29.355567669 -0500
@@ -138,7 +138,7 @@
 .SH "USAGE"
 .IX Header "USAGE"
 Usage:
-  policyd-spf [/etc/policyd-spf/policyd-spf.conf]
+  policyd-spf [/etc/postfix-policyd-spf-python/policyd-spf.conf]
 
 .SH "OTHER DOCUMENTATION"
 .IX Header "OTHER DOCUMENTATION"
@@ -166,7 +166,7 @@
 
 Configuration options are described here and in the configuration file 
 provided with the package.  The provided setup.py installs this configuration 
-file in /etc/policyd-spf/.
+file in /etc/postfix-policyd-spf-python/.
 
 Additionally, whitelisting certain IP addresses from SPF checks is supported.
 This man page and the sample configuration file show the format to use.
Index: pypolicyd-spf-2.0.0/setup.py
===================================================================
--- pypolicyd-spf-2.0.0.orig/setup.py	2016-12-02 01:23:29.363567669 -0500
+++ pypolicyd-spf-2.0.0/setup.py	2016-12-02 01:23:29.355567669 -0500
@@ -15,9 +15,9 @@
       scripts = ['policyd-spf'],
       data_files=[(os.path.join('share', 'man', 'man1'),
           ['policyd-spf.1']), (os.path.join('share', 'man', 'man5'),
-          ['policyd-spf.conf.5']), (os.path.join('/etc', 'python-policyd-spf'),
-          ['policyd-spf.conf']), (os.path.join('share', 'man', 'man5'),
-          ['policyd-spf.peruser.5'])],
+          ['policyd-spf.conf.5']), (os.path.join('../etc',
+          'postfix-policyd-spf-python'), ['policyd-spf.conf']),
+          (os.path.join('share', 'man', 'man5'), ['policyd-spf.peruser.5'])],
       classifiers = [
         'Development Status :: 5 - Production/Stable',
         'Environment :: No Input/Output (Daemon)',
